keyboards/svalboard/keymaps/keymap_support.c

```diff
@@ -271,8 +277,14 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
                 mouse_mode(false);
         }
     }
-    if (record->event.pressed) {
+    if (record->event.pressed) { // key pressed
         switch (keycode) {
+            case SV_LEFT_SCROLL_HOLD:
```

Comment:
Don't modify global_saved_values without calling write_eeprom_kb().

Make another global variable and, | the two values together.

Reply:
Done